team_comparison_page.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Team comparison page</title>
+    <meta name="description" content="This is a page where you can compare overall strength of chosen teams">
+</head>
+<body>
+    <h1 align="center">Compare teams</h1>
+    <table>
+        <tr><h4>Competition</h4></tr>
+        <tr><h3><a href="team_page.html">Team name</a></h3></tr>
+        <tr>
+            <button>Choose prev team</button>
+            <img src="tu wstaw ścieżkę do obrazka" alt="Team crest">
+            <button>Choose next team</button>
+        </tr>
+    </table>
+    <table><h2><b>VS</b></h2></table>
+    <table>
```

Comment:
I see you using pretty often Table tag. What's the reason on this? The table has a semantic meaning that data inside it are should be treated as tables. If it's not like this please use divs.